src/js/layout/Header.js

```diff
@@ -0,0 +1,18 @@
+import { $createElement } from "../utils/dom.js";
+import logo from "../../images/logo.png";
+
+export const Header = () => {
+  const $headerElement = $createElement("header");
+
+  $headerElement.innerHTML = `
+    <h1>
+      <img src=${logo} alt="MovieList 로고" />
+    </h1>
+    <div class="search-box">
+      <input type="text" placeholder="검색" />
+      <button class="search-button">검색</button>
+    </div>
+  `;
+
+  return $headerElement;
```

Comment:
```suggestion
  return $createElement(`
    <header>
      <h1>
        <img src=${logo} alt="MovieList 로고" />
      </h1>
      <div class="search-box">
        <input type="text" placeholder="검색" />
        <button class="search-button">검색</button>
      </div>
    </header>
  `);
```
이렇게 표현할 수 있도록 createElement 함수를 개선해보면 어떨까요!?